rpm/gratia-probe.spec

```diff
@@ -258,19 +258,30 @@ git_commit_id=$(gzip -d < %{SOURCE0} | git get-tar-commit-id)
   install -d $RPM_BUILD_ROOT/%{_sysconfdir}/condor/config.d
   install -m 644 condor/99_gratia.conf $RPM_BUILD_ROOT/%{_sysconfdir}/condor/config.d/99_gratia.conf
   install -m 644 condor/99_gratia-gwms.conf $RPM_BUILD_ROOT/%{_sysconfdir}/condor/config.d/99_gratia-gwms.conf
+  rm $RPM_BUILD_ROOT%{_datadir}/gratia/condor/99_gratia.conf
   rm $RPM_BUILD_ROOT%{_datadir}/gratia/condor/99_gratia-gwms.conf
 
   # Install the htcondor-ce configuration
   install -d $RPM_BUILD_ROOT/%{_sysconfdir}/condor-ce/config.d
-  install -m 644 condor/99_gratia.conf $RPM_BUILD_ROOT/%{_sysconfdir}/condor-ce/config.d/99_gratia.conf
-  rm $RPM_BUILD_ROOT%{_datadir}/gratia/condor/99_gratia.conf
-  install -m 644 condor/gratia-probe-htcondor-ce.cron $RPM_BUILD_ROOT/%{_sysconfdir}/cron.d/gratia-probe-htcondor-ce.cron
+  install -m 644 condor/99_gratia-ce.conf $RPM_BUILD_ROOT/%{_sysconfdir}/condor-ce/config.d/99_gratia-ce.conf
   install -d $RPM_BUILD_ROOT%{_datadir}/gratia/htcondor-ce/
   install -m 755 condor/condor_meter $RPM_BUILD_ROOT%{_datadir}/gratia/htcondor-ce/
   # Copy the condor configuration
   install -d $RPM_BUILD_ROOT/%{_sysconfdir}/gratia/htcondor-ce
   install -m 644 $RPM_BUILD_ROOT/%{_sysconfdir}/gratia/condor/ProbeConfig $RPM_BUILD_ROOT/%{_sysconfdir}/gratia/htcondor-ce/ProbeConfig
+
+  update_probeconfig () {
+    sed -i "s|$2=\"[^\"]*\"|$2=\"$3\"" \
+        $RPM_BUILD_ROOT/%{_sysconfdir}/gratia/$1/ProbeConfig
+  }
+
+  update_probeconfig htcondor-ce WorkingFolder /var/lib/condor-ce
+  update_probeconfig htcondor-ce LogFolder     /var/log/condor-ce
+  update_probeconfig htcondor-ce Lockfile      /var/lock/condor-ce/gratia.lock
   sed -i 's/ProbeName="condor:/ProbeName="htcondor-ce:/' $RPM_BUILD_ROOT/%{_sysconfdir}/gratia/htcondor-ce/ProbeConfig
+  sed -i '/WorkingFolder="/s|"[^"]*"|"/var/lib/condor-ce|"' $RPM_BUILD_ROOT/%{_sysconfdir}/gratia/htcondor-ce/ProbeConfig
+  sed -i '/LogFolder="/s|"[^"]*"|"/var/log/condor-ce|"' $RPM_BUILD_ROOT/%{_sysconfdir}/gratia/htcondor-ce/ProbeConfig
+  sed -i '/LogFolder="/a\    Lockfile="/var/lock/condor-ce/gratia.lock"' $RPM_BUILD_ROOT/%{_sysconfdir}/gratia/htcondor-ce/ProbeConfig
```

Comment:
```suggestion
```

I think we can drop these with your most recent commit

Reply:
I had some extra cruft left in by accident due to a `git commit -am ...` when i just meant `-m`

anyway i've cleaned this up since